Agreements/Internal/TCCOW_TCNW.txt

```diff
@@ -10,7 +10,7 @@ COPX:*:*:SOPIT:EGLL:*:London TC COWLY:London TC NW:*:15000:SOPIT
 
 COPX:*:*:SOPIT:EGWU:*:London TC COWLY:London TC NW:*:15000:SOPIT5
 
-COPX:*:*:CLIPY:EGMC:*:London TC COWLY:London TC NW:*:15000:CLIPY
```

Comment:
Same as Essex, until archived STARs removed, we#ll need to keep this

Reply:
changing clipy agreements in another PR